### ISSUE TITLE
Fix metric data loading

### ISSUE THREAD

**Issue by georgf** [mozilla/mdv2-prototype#54]
We currently load data from `/data/...`:
```javascript
const response = await fetch(`/data/${metric}_${channel}_${version}.json`);
```

This breaks e.g. with github pages, which has the data under `/mdv2/data/...`.

**Comment by georgf** [comment]
We could for now just use the workaround of fetching from "current URL" + "/data/...".

**Comment by chutten** [comment]
If `fetch` is implemented the way I think it is we may be able to simply remove the initial `/` from the url and it will turn into one relative to the current path.

**Comment by ecomerford** [comment]
Yep, that works. I'll go ahead and push those changes through now.

**Comment by ecomerford** [comment]
Solved by #55.
